src/frontend/Komponenter/Oppgavebenk/OppgavebenkApp.tsx

```diff
@@ -38,11 +51,37 @@ export const OppgavebenkApp: React.FC = () => {
                 method: 'POST',
                 url: `/familie-ef-sak/api/oppgave/soek`,
                 data,
-            }).then((res: Ressurs<IOppgaverResponse>) => settOppgaveRessurs(res));
+            }).then((res: Ressurs<IOppgaverResponse>) => {
+                settOppgaveRessurs(res);
+                if (res.status === RessursStatus.SUKSESS) {
+                    settOppgaveRespons(res.data);
+                }
+            });
         },
         [axiosRequest]
     );
 
+    const oppdaterOppgave = (oppgaveId: string) => {
```

Comment:
Nesten så oppgave burde få sin egen context :-) 
Men tenker også at fordel oppgave burde kunne returnere oppdatert oppgave, så trengs ikke denne (men trenger fortsatt å oppdatere oppgaven som man har fordelt)

Reply:
Ja, vi diskuterte dette og landet på at det foreløpig ikke var behov for det. Men vi kan jo vurdere når vi utvider funksjonaliteten senere :)